futurenet/soroban-rpc/etc/soroban-rpc.env

```
@@ -4,3 +4,11 @@ export ENDPOINT=localhost:8003
 export HORIZON_URL=http://localhost:8001
 export NETWORK_PASSPHRASE="__NETWORK__"
 export STELLAR_CORE_URL="http://localhost:11626"
+export CAPTIVE_CORE_CONFIG_PATH=/opt/stellar/soroban-rpc/etc/stellar-captive-core.cfg
+export CAPTIVE_CORE_STORAGE_PATH=/opt/stellar/soroban-rpc/cc
```

Does this directory path need updating? I see further down the directory created is named `captive-core`.
```suggestion
export CAPTIVE_CORE_STORAGE_PATH=/opt/stellar/soroban-rpc/captive-core
```

---

nice catch, it was created on the fly which didn't cause runtime problem, but this was the intention, thanks - https://github.com/stellar/quickstart/pull/414/commits/60745c69668a644952b20f595febebd9ef479e45